cockatrice/src/dlg_forgotpassword.cpp

```diff
@@ -0,0 +1,67 @@
+#include <QLabel>
+#include <QGridLayout>
+#include <QHBoxLayout>
+#include <QDialogButtonBox>
+#include <QMessageBox>
+#include <QDebug>
+
+#include "dlg_forgotpassword.h"
+
+DlgForgotPassword::DlgForgotPassword(QWidget *parent)
+    : QDialog(parent)
+{
+    hostLabel = new QLabel(tr("Host:"));
+    hostEdit = new QLineEdit();
+
+    portLabel = new QLabel(tr("Port:"));
+    portEdit = new QLineEdit();
+
+    playernameLabel = new QLabel(tr("Player Name:"));
+    playernameEdit = new QLineEdit();
+
+    emailLabel = new QLabel(tr("Email:"));
+    emailEdit = new QLineEdit();
+
+	QGridLayout *grid = new QGridLayout;
+	grid->addWidget(hostLabel, 0, 0);
+	grid->addWidget(hostEdit, 0, 1);
+	grid->addWidget(portLabel, 1, 0);
+	grid->addWidget(portEdit, 1, 1);
+	grid->addWidget(playernameLabel, 2, 0);
+	grid->addWidget(playernameEdit, 2, 1);
+	grid->addWidget(emailLabel, 3, 0);
+	grid->addWidget(emailEdit, 3, 1);
+
+    QDialogButtonBox *buttonBox = new QDialogButtonBox(QDialogButtonBox::Ok | QDialogButtonBox::Cancel);
```

Comment:
When OK is pressed, the dialog will close no matter what and if you had something wrong in the details will have to start all over again... maybe persist the data or auto-reopen the dialog in case of failure?